packages/scss/demo/templates/side-cards.html

```diff
@@ -11,27 +11,27 @@
 	</head>
 	<body>
 		<main class="main">
-			<aside class="navSide">
-				<a href="#" class="navSide-link is-active">
+			<aside class="navSide mod-compact">
+				<a href="#" class="navSide-menuItem-item is-active">
```

Comment:
This element is named like a child of a child. But it's only a direct child of navSide, so the class name must be wrong.